bin/prepare_project_edition.sh

```diff
@@ -36,15 +39,27 @@ echo "> Move ${DEPENDENCY_PACKAGE_DIR} to ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACK
 mkdir -p ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}
 mv ${DEPENDENCY_PACKAGE_DIR}/* ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}/
 
+# Remove installed dependencies inside the package
+rm -rf ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}/vendor
```

Comment:
In `ezcommerce-base-design` and `ezcommerce-erp-admin` the packages installed inside the tested package interefered with the whole setup (causing duplicated webpack encore entries).